packages/es-components/src/components/containers/notification/Notification.js

```diff
@@ -268,6 +276,14 @@ Notification.propTypes = {
   callsToAction: PropTypes.arrayOf(
     PropTypes.oneOfType([PropTypes.shape(callToActionShape), PropTypes.element])
   ),
+  /**
+   * Use the light color variants of the alert
+   */
+  light: PropTypes.bool,
```

Comment:
I'd prefer these two bools be named like a statement, something like useLightVariant or isLightVariant.